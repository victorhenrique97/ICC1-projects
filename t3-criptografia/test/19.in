8.bin
2
