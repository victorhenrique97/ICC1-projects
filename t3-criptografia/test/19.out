Bolacha ou biscoito? Resposta correta: bolacha! Voce nao vai acertar esse caso de teste se o seu programa responder biscoito HAHAHAHAHAHAHAHAHAHAHAHAHAHAHAHAHAHAHAHAHAH.
